serverless/knative_eventing/serverless-kafka.adoc

```diff
@@ -7,6 +7,9 @@ include::modules/common-attributes.adoc[]
 toc::[]
 
 You can use the `KafkaChannel` channel type and `KafkaSource` event source with {ServerlessProductName}.
+
+Using Kafka components in your event-driven architecture provides _at least once_ guarantees for event delivery. This means that operations are retried until a return code value is received. However, while this makes your application more resilient to lost events, it may result in duplicate events being sent.
```

Comment:
I might be parsing the first sentence incorrectly, but should "... architecture provides at least once guarantees" be either:

"... architecture provides at least one guarantee"

or

"... architecture at least once guarantees"

?

Reply:
"at least once" is a term that means something specific, i.e. it's a type of guarantee, so I'd leave as is 🙂